logger/logOutputSubject.go

```diff
@@ -5,7 +5,7 @@ import (
 	"io"
 	"sync"
 
-	protobuf "github.com/ElrondNetwork/elrond-go/logger/proto"
```

Comment:
+1 for wrapping the inside of the marshalizer so that the subject does not know about the internals

Reply:
Yeah, finally found the proper way to deal with all 3 marshalizers and having a cleaner, comprehensible implementation.